src/iden3comm/types/protocol/auth.ts

```diff
@@ -1,6 +1,6 @@
 import { ZKProof } from '@iden3/js-jwz';
 import { MediaType } from '../../constants';
-import { JSONObject, ProtocolMessage } from '../index';
+import { JSONObject, ProtocolMessage } from '../packer';
```

Comment:
Suggestion: since these types are used in the protocol message types, SDK clients may need to reference them. It would be clearer to have them in a separate `types.ts` file, rather than define them in packer. The `index.ts` should still re-export them.